backend/apps/core/models.py

```
@@ -202,5 +202,112 @@ class AuditableModel(models.Model):
+    Use ``TenantModel.objects.unscoped()`` to bypass scoping for
+    admin/migration code (rarely needed in request paths).
+    """
+
+    def get_queryset(self):
+        qs = TenantQuerySet(self.model, using=self._db)
+        org_id = None
+        try:
+            from apps.core.tenant import get_current_tenant_id
+
+            org_id = get_current_tenant_id()
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)